### ISSUE TITLE
Blogs on Stacks 

### ISSUE THREAD

**Issue by krn17** [stacksgov/Stacks-Grant-Launchpad#274]
**Background**
`What problems do you aim to solve? How does it serve the mission of a user owned internet?`

I want to solve an inefficiency, instead of a problem. I want to build more content on Stacks and its ecosystem, in an easy to understand language, for the non-tech and newbies in the blockchain space. The more people know the value Stacks provides, the better it is for the ecosystem and furthers the mission of user internet owned.

**Project Overview**
`What solution are you providing? Who will it serve?`
I want to write a series of Medium blogs on Stacks along with my co-writer. Theses articles would be covering various aspects of decentralisation and Stacks in Layman's language. 

As more and more people are on-boarding on various blockchains, the curiosity to learn and invest in missions served by these chains emerge. Yet, the documentations and most content on the web usually includes a lot of tech jargon, making it hard to understand for the non-tech audience. I believe in Stacks' mission and would love to help/assist the chain by helping it reach the masses through my content.

**Scope**
`What are the components or technical specs of the project? What will the final deliverable look like? How will you measure success?`

As per my discussions with Kenny and Bill (on discord and email), I would love write on their suggested topics -

1. A comparison of PoW, PoS, and PoX consensus protocols, with their pros and cons outlined

2. A deep dive into PoX consensus protocol with the aim of making it easy to understand

3. Why Stacks is good for Bitcoin - a look at why bringing smart contracts to Bitcoin will help the Bitcoin economy grow and provide new use cases for Bitcoin, without actually having to modify Bitcoin itself at all. Ideally would include the limitations of Bitcoin Script and how a smart contract language/platform like Clarity/Stacks adds needed functionality.

4. Potential use cases for DeFi on Bitcoin 

**Budget and Milestones**
`What grant amount are you seeking? How long will the project take in hours? If more than 20, please break down the project into milestones, with a clear output (e.g., low-fi mockup, MVP with two features) and include the estimated work hours for each milestone.`

Bill suggested an ideal breakdown and cost for the same and I'd love to follow the same. It is - 

Total Grant Request: $5000

M1: For Milestone 1, I would start with the first topic - _A comparison of PoW, PoS, and PoX consensus protocols, with their pros and cons outlined_. For this piece, a grant of **$1.5k** would suffice and if after reviewing the work quality everything seems good, then we can move on to M2.

M2: For Milestone 2, I would complete the rest of the listed article topics - 2,3,4. And for the same, a budget of $3.5k would be requested.

Thus totaling to $5k for the complete project.

**Team**
`Who is building this? What relevant experience do you bring to this project? Are there skills sets you are missing that you are seeking from the community? Please share links to previous work. `

I'd be working on majority of it, taking some assistance and help from my other co-writer (discord - Shivani#6082). We've previously published several articles and blogs and have good experience writing technical pieces in layman's' language.

**Risks**
`What dependencies or obstacles do you anticipate? What contingency plans do you have in place?`

Time. This is an effort intensive project and writing good quality articles requires time. I'd try to complete the article pieces at a moderate pace, retaining the quality of the content.

**Community and Supporting Materials**
`Do you have previous projects, code commits, or experiences that are relevant to this application? What community feedback or input have you received? How do you plan to share your plan to the community over time and as the final deliverable?`
N/A

**Comment by tycho1212** [comment]
Interesting.

I'd be curious if there's a specific audience you want to target with the articles and are there any expectations around how many people you'll reach/how? In my experience, writing content is one thing and making sure its read is another. Do you have an audience yourself?

Some supporting materials about your experience feels important for this kind of grant

**Comment by krn17** [comment]
> Interesting.
> 
> I'd be curious if there's a specific audience you want to target with the articles and are there any expectations around how many people you'll reach/how? In my experience, writing content is one thing and making sure its read is another. Do you have an audience yourself?
> 
> Some supporting materials about your experience feels important for this kind of grant

Hey @tycho1212, let me address your queries in these points - 

1. Not targeting any specific audience, but since I'd be aiming to write the blogs in easy to understand language, I believe it can serve as a good starting point for those who come from non-tech and with relatively less blockchain knowledge. 
2. Your point on the reader's part is very valid, and yes we have a few readers, but not many. Me and co-writer publish from her blog account which is also a part of Crypto Stars publication on Medium. This Publication receives 1600+ views daily and each morning all the subscribers receive the daily digest with the stories published in the previous day. We'd publish the Stacks blogs and include our submissions in this publication as well.
3. Moreover, we may not be able to bring hundreds or thousands of readers to the blogs immediately, but I believe as the Stacks' adaptation grows further, these blogs can serve as a good resource for getting started. If the work quality seems good, perhaps we may publish more on Stacks and these articles could rank high in search engines with time serving as good starting points.
4. Extending the last point, if the blogs need to be SEO optimized, it'd require more work and research. For instance, an average well-written Medium article is about 6-8 mins read, a better one is almost an 8 mins read at 2000+ words, which is pretty detailed.
5. If we write more articles on Stacks moving ahead, I think we'd be able to do good SEO by interlinking withing these blogs, which would help rank on Google as well. The off-page SEO can be facilitated by the Stacks community by sharing these articles.

Supporting material - consider this [article ](https://blog.cryptostars.is/a-detailed-guide-to-biconomy-relayer-protocol-for-seamless-and-gasless-ux-9e50601114ea), which is a 16 min read on Biconomy Protocol. This piece was written by me and my co-writer few days back.

I had not taken SEO into consideration in my proposal, but if that's an interest, we might have to increase the budget a little. Since then the content needs to be more SEO conscious. Read more on Medium posts SEO [here](https://medium.com/menlo-office/medium-seo-how-to-rank-your-medium-articles-in-google-search-e7bdd8d962c0).

Open to your thoughts @tycho1212

**Comment by will-at-stacks** [comment]
Hi @krn17  - Please excuse us if your review takes a bit longer than normal as e are in the process of rolling out a new platform.  Additionally, I am out of the office until 2/7.  We will get back to you as soon as we can.  Best, Will

**Comment by krn17** [comment]
Sure @will-at-stacks, no hurry at my end :)

**Comment by JakeBlockchain** [comment]
I like the idea of one single piece of content to point to that compares PoW, PoX and PoS in a clear, concise manner.

Like Tycho, I have questions about distribution and audience. As a content creator myself, I have definitely underestimated how difficult it is to get the content you create in front of the right people and have them engage with it.
I see the medium post you linked to have a good amount of likes since its been posted.

**Comment by krn17** [comment]
@JakeBlockchain love your podcasts dude! and thanks for the comment :)
Feel free to shoot any questions you might have

**Comment by will-at-stacks** [comment]
@krn17 
Thank you for the proposal.  We are very much in favor of supporting anything that promotes Bitcoin / Stacks education.  Since we are new to working with you we would like you to consider a revised proposal - we would like to cut this in half (2) articles for $2500 and see how things go.  The two topics we feel will travel the furtherest are:

- Why Stacks is good for Bitcoin - a look at why bringing smart contracts to Bitcoin will help the Bitcoin economy grow and provide new use cases for Bitcoin, without actually having to modify Bitcoin itself at all. Ideally would include the limitations of Bitcoin Script and how a smart contract language/platform like Clarity/Stacks adds needed functionality.

- Potential use cases for DeFi on Bitcoin

These are the articles we would like you to start with.

Please let me know if this sounds agreeable.

Thanks,
Will

**Comment by krn17** [comment]
Hi @will-at-stacks, these are good topics to start with, sure let's do this :)

I'll start with the above mentioned topics asap and keep you updated.

**Comment by will-at-stacks** [comment]
@krn17 glad this works for you!  I will mark this as approved and get your funds disbursed.

**Comment by will-at-stacks** [comment]
Hi @krn17  just checking in.  Can you please provide us with an update on this grant?  Thanks, Will

**Comment by krn17** [comment]
Hey @will-at-stacks I've completed a considerable part of both the pieces and would probably be able to finish them before the next Monday.
At this moment I'm also involved in organizing a considerably big web3 IRL meetup and ensuring everything's going good.
As for the articles I'm researching on Stacks in depth, not just for the writing part, but to be an important contributor in the ecosystem myself  :)

Be assured, you'd love the work. I'll keep you updated.

**Comment by Akibalogh** [comment]
Hi @krn17 - we're writing a few blog posts on Stacks and use cases like Bitcoin lending.  Makki from our team is working on our first blog post.  

We'd love to coordinate with you -- please friend me on Discord?  akibalogh#1043

**Comment by krn17** [comment]
hey @Akibalogh sounds good, dropped a request to you

**Comment by krn17** [comment]
hey @will-at-stacks is it fine if I submit the blogs in April?

**Comment by will-at-stacks** [comment]
Hello and thank you for participating in the Stacks Foundation Grants Program!

We are in the process of migrating from GitHub to the new [Grants Dashboard](https://grants.stacks.org/). In order to complete your grant, you will need to submit any remaining Progress Review and/or Final Review requests through the Dashboard in order to receive your remaining payments.

- If you have questions about the Grants Dashboard in general please email [ambassadors@stacks.org](mailto:ambassadors@stacks.org).
- If you have questions regarding payments and/or contracts please email [payments@stacks.org](mailto:payments@stacks.org).

Lastly, please note we are marking this grant 'closed' on GitHub for organizational purposes, but it is still 'open' on the Grants Dashboard.

Thanks and we hope to continue to support your efforts with additional grants!

Best,
Will
